src/CoreBundle/Resources/public/javascript/modules/sidebar/sidebar.js

```diff
@@ -201,6 +220,32 @@
             $.post(url, {
                 clickedMenuId: clickedMenuId
             });
+        },
+        extractTableId: function(url) {
+            // TODO use some proper url splitting? ie https://gomakethings.com/getting-all-query-string-values-from-a-url-with-vanilla-js/
```

Comment:
There is a lot easier way in all browsers except IE11 (which would be o.k.): https://caniuse.com/#search=URLSearchParams
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams
https://www.sitepoint.com/get-url-parameters-with-javascript/

We should iterate over all parameters and check for keys not only with the exact pagedef name, but allow tablemanager* and tableeditor* because there may be some extensions in customer projects or even in the core using a different tablemanager.

Reply:
Ok, using URLSearchParams. The substring search for parameters is already present.

However there are still other cases not covered. A direct comparisson of links should get them though.